modules/client-configuration/pages/virtualization.adoc

```diff
@@ -10,22 +10,45 @@ The range of capabilities that virtualized clients have depends on the third-par
 Xen and KVM hosts and guests can be managed directly in {productname}.
 This enables you to autoinstall hosts and guests using {ay} or {kickstart}, and manage guests in the {webui}.
 
+Additionally, {caasp} clusters can be managed in the {productname} {webui}, by navigating to menu:Clusters[Overview].
+
 For VMWare, including VMWare vSphere, and Nutanix AHV, {productname} requires you to set up a virtual host manager (VHM) to control the VMs.
 This gives you control over the hosts and guests, but in a more limited way than available with Xen and KVM.
 
-
 //So I looked it up in their docs: "VMWare vSphere is a suite of virtualization applications that includes ESXi and vCenter Server". So I think using "VMWare vSphere" implies ESXi and vCenter without having to spell them out. Happy to be proven wrong. --LKB 2019-07-10
 
 Other third-party virtualization providers are not directly supported by {productname}.
 However, if your provider allows you to export a JSON configuration file for the VM, you can upload that configuration file to {productname} and manage it with a VHM.
 
+For more information about using VHMs to manage virtualization, see xref:client-configuration:vhm.adoc[].
+
+
+
+
```

Comment:
I think you can ifdef this section for suma-content.

Reply:
Oh, good idea.